src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportFileEtagNotMatchException.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Import
+{
+    public class ImportFileEtagNotMatchException : Exception
+    {
+        public ImportFileEtagNotMatchException(string message)
+            : base(message)
+        {
+            EnsureArg.IsNotNull(message, nameof(message));
```

Comment:
>EnsureArg.IsNotNull [](http://example.com/codeflow?start=12&length=19)

Exceptions should use Debug.Assert. Else we risk throwing another exception

Reply:
Thanks, updated.